<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>638</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>638</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>638</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>400</width>
     <height>638</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>620</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>620</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0.960894, y1:0.062, x2:0.095, y2:0.977909, stop:0 rgba(255, 0, 0, 255), stop:0.346369 rgba(255, 179, 179, 255), stop:1 rgba(0, 85, 255, 255));</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>40</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QPushButton" name="boton_cerrar">
          <property name="geometry">
           <rect>
            <x>340</x>
            <y>10</y>
            <width>41</width>
            <height>24</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">
QPushButton{
background-color: transparent;
color: rgb(255, 255, 255);

font: 24pt &quot;Copperplate Gothic Bold&quot;;
}

QPushButton:hover{
background-color: rgba(0, 0, 0, 30);
color: rgb(0, 0, 0);
}

QPushButton:pressed{
background-color: rgba(0, 0, 0, 60);
border-radius: 20px;
}</string>
          </property>
          <property name="text">
           <string>X</string>
          </property>
         </widget>
         <widget class="QPushButton" name="boton_minimizar">
          <property name="geometry">
           <rect>
            <x>290</x>
            <y>10</y>
            <width>41</width>
            <height>24</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">
QPushButton{
background-color: transparent;
color: rgb(255, 255, 255);

font: 24pt &quot;Copperplate Gothic Bold&quot;;
}

QPushButton:hover{
background-color: rgba(0, 0, 0, 30);
color: rgb(0, 0, 0);
}

QPushButton:pressed{
background-color: rgba(0, 0, 0, 60);
border-radius: 20px;
}</string>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>0</y>
            <width>81</width>
            <height>41</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: transparent;
color: rgb(255, 255, 255);
</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;Obvio&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_result">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">
QLabel{
font: 87 12pt &quot;Arial Black&quot;;
background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
border: 1px solid #14c8dc;

}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_prev">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial Black</family>
           <pointsize>12</pointsize>
           <weight>10</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">
QLabel{
font: 87 12pt &quot;Arial Black&quot;;
background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
border: 1px solid #14c8dc;

}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="2" column="3">
           <widget class="QPushButton" name="boton_multiplicar">
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>80</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
background-color: transparent;
border-radius: 20px;
font: 24pt &quot;Copperplate Gothic Bold&quot;;
}

QPushButton:hover{
background-color: rgba(0, 0, 0, 30);
border-radius: 20px;
}

QPushButton:pressed{
background-color: rgba(0, 0, 0, 60);
border-radius: 20px;
}</string>
            </property>
            <property name="text">
             <string>*</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QPushButton" name="boton_dividir">
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>80</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
background-color: transparent;
border-radius: 20px;
font: 24pt &quot;Copperplate Gothic Bold&quot;;
}

QPushButton:hover{
background-color: rgba(0, 0, 0, 30);
border-radius: 20px;
}

QPushButton:pressed{
background-color: rgba(0, 0, 0, 60);
border-radius: 20px;
}</string>
            </property>
            <property name="text">
             <string>/</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="boton_09">
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>80</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
background-color: transparent;
border-radius: 20px;
font: 24pt &quot;Copperplate Gothic Bold&quot;;
}

QPushButton:hover{
background-color: rgba(0, 0, 0, 30);
border-radius: 20px;
}

QPushButton:pressed{
background-color: rgba(0, 0, 0, 60);
border-radius: 20px;
}</string>
            </property>
            <property name="text">
             <string>9</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="boton_01">
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>80</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
background-color: transparent;
border-radius: 20px;
font: 24pt &quot;Copperplate Gothic Bold&quot;;
}

QPushButton:hover{
background-color: rgba(0, 0, 0, 30);
border-radius: 20px;
}

QPushButton:pressed{
background-color: rgba(0, 0, 0, 60);
border-radius: 20px;
}</string>
            </property>
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QPushButton" name="boton_sumar">
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>80</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
background-color: transparent;
border-radius: 20px;
font: 24pt &quot;Copperplate Gothic Bold&quot;;
}

QPushButton:hover{
background-color: rgba(0, 0, 0, 30);
border-radius: 20px;
}

QPushButton:pressed{
background-color: rgba(0, 0, 0, 60);
border-radius: 20px;
}</string>
            </property>
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="boton_05">
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>80</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
background-color: transparent;
border-radius: 20px;
font: 24pt &quot;Copperplate Gothic Bold&quot;;
}

QPushButton:hover{
background-color: rgba(0, 0, 0, 30);
border-radius: 20px;
}

QPushButton:pressed{
background-color: rgba(0, 0, 0, 60);
border-radius: 20px;
}</string>
            </property>
            <property name="text">
             <string>5</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="boton_03">
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>80</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
background-color: transparent;
border-radius: 20px;
font: 24pt &quot;Copperplate Gothic Bold&quot;;
}

QPushButton:hover{
background-color: rgba(0, 0, 0, 30);
border-radius: 20px;
}

QPushButton:pressed{
background-color: rgba(0, 0, 0, 60);
border-radius: 20px;
}</string>
            </property>
            <property name="text">
             <string>3</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QPushButton" name="boton_restar">
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>80</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
background-color: transparent;
border-radius: 20px;
font: 24pt &quot;Copperplate Gothic Bold&quot;;
}

QPushButton:hover{
background-color: rgba(0, 0, 0, 30);
border-radius: 20px;
}

QPushButton:pressed{
background-color: rgba(0, 0, 0, 60);
border-radius: 20px;
}</string>
            </property>
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QPushButton" name="boton_igual">
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>80</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
background-color: transparent;
border-radius: 20px;
font: 24pt &quot;Copperplate Gothic Bold&quot;;
}

QPushButton:hover{
background-color: rgba(0, 0, 0, 30);
border-radius: 20px;
}

QPushButton:pressed{
background-color: rgba(0, 0, 0, 60);
border-radius: 20px;
}</string>
            </property>
            <property name="text">
             <string>=</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="boton_02">
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>80</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
background-color: transparent;
border-radius: 20px;
font: 24pt &quot;Copperplate Gothic Bold&quot;;
}

QPushButton:hover{
background-color: rgba(0, 0, 0, 30);
border-radius: 20px;
}

QPushButton:pressed{
background-color: rgba(0, 0, 0, 60);
border-radius: 20px;
}</string>
            </property>
            <property name="text">
             <string>2</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="boton_07">
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>80</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
background-color: transparent;
border-radius: 20px;
font: 24pt &quot;Copperplate Gothic Bold&quot;;
}

QPushButton:hover{
background-color: rgba(0, 0, 0, 30);
border-radius: 20px;
}

QPushButton:pressed{
background-color: rgba(0, 0, 0, 60);
border-radius: 20px;
}</string>
            </property>
            <property name="text">
             <string>7</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="boton_08">
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>80</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
background-color: transparent;
border-radius: 20px;
font: 24pt &quot;Copperplate Gothic Bold&quot;;
}

QPushButton:hover{
background-color: rgba(0, 0, 0, 30);
border-radius: 20px;
}

QPushButton:pressed{
background-color: rgba(0, 0, 0, 60);
border-radius: 20px;
}</string>
            </property>
            <property name="text">
             <string>8</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="boton_04">
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>80</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
background-color: transparent;
border-radius: 20px;
font: 24pt &quot;Copperplate Gothic Bold&quot;;
}

QPushButton:hover{
background-color: rgba(0, 0, 0, 30);
border-radius: 20px;
}

QPushButton:pressed{
background-color: rgba(0, 0, 0, 60);
border-radius: 20px;
}</string>
            </property>
            <property name="text">
             <string>4</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QPushButton" name="boton_0">
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>80</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
background-color: transparent;
border-radius: 20px;
font: 24pt &quot;Copperplate Gothic Bold&quot;;
}

QPushButton:hover{
background-color: rgba(0, 0, 0, 30);
border-radius: 20px;
}

QPushButton:pressed{
background-color: rgba(0, 0, 0, 60);
border-radius: 20px;
}</string>
            </property>
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="boton_clean">
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>80</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
background-color: transparent;
border-radius: 20px;
font: 24pt &quot;Copperplate Gothic Bold&quot;;
}

QPushButton:hover{
background-color: rgba(0, 0, 0, 30);
border-radius: 20px;
}

QPushButton:pressed{
background-color: rgba(0, 0, 0, 60);
border-radius: 20px;
}</string>
            </property>
            <property name="text">
             <string>C</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="boton_06">
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>80</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
background-color: transparent;
border-radius: 20px;
font: 24pt &quot;Copperplate Gothic Bold&quot;;
}

QPushButton:hover{
background-color: rgba(0, 0, 0, 30);
border-radius: 20px;
}

QPushButton:pressed{
background-color: rgba(0, 0, 0, 60);
border-radius: 20px;
}</string>
            </property>
            <property name="text">
             <string>6</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
